collect_app/src/main/java/org/odk/collect/android/application/initialization/GoogleDriveProjectsDeleter.kt

```diff
@@ -19,7 +21,15 @@ class GoogleDriveProjectsDeleter(
             val protocol = unprotectedSettings.getString(ProjectKeys.KEY_PROTOCOL)
 
             if (protocol == ProjectKeys.PROTOCOL_GOOGLE_SHEETS) {
-                projectDeleter.deleteProject(it.uuid)
+                // try to delete
+                val result = projectDeleter.deleteProject(it.uuid)
+
+                // if project cannot be deleted then convert it to ODK protocol
+                if (result == DeleteProjectResult.UnsentInstances || result == DeleteProjectResult.RunningBackgroundJobs) {
+                    unprotectedSettings.save(ProjectKeys.KEY_PROTOCOL, ProjectKeys.PROTOCOL_SERVER)
+                    unprotectedSettings.save(ProjectKeys.KEY_SERVER_URL, "https://example.com")
+                    settingsProvider.getMetaSettings().save(MetaKeys.getKeyIsOldGDProject(it.uuid), true)
```

Comment:
Why save this to meta settings and not just to normal ("unprotected") settings?

Reply:
My thinking was that unprotected settings represent the section of unprotected preferences in the app, the same with the protected ones and if we want to save something that does not have a visual representation then it should be in meta settings.

Reply:
I'd more thought that anything that's a "project" level goes in "unprotected". I guess this is really information "about" the project itself rather than a setting so I do see why it's in meta, but it feels a little awkward that you have to still index by project. Would it be a possibility that this could live on `Project` itself?

Reply:
> Would it be a possibility that this could live on Project itself?

Yeah, I think it's a good idea.